@navikt/core/react/src/form/combobox/FilteredOptions/filtered-options-util.ts

```diff
@@ -19,9 +19,7 @@ const getFilteredOptionsId = (comboboxId: string) =>
   `${comboboxId}-filtered-options`;
 
 const getOptionId = (comboboxId: string, option: string) =>
-  `${comboboxId.toLocaleLowerCase()}-option-${option
-    .replace(" ", "-")
-    .toLocaleLowerCase()}`;
+  `${comboboxId}-option-${option}`.replace(/\s/g, "-").toLocaleLowerCase();
```

Comment:
Did a minimal fix here, but could rewrite more of these functions to also take account for comboboxId potentially including space?

Reply:
I don't think we need to do that.